### PR TITLE
Added a Template for a Movie Website

### PR DESCRIPTION
**Pull request by RajAdwaita**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

I created a frontend template using basic HTML CSS js and bootstrap and added it to this repository. This is a movie based website that also has a movie success prediction model in it.  

Fixes # (issue no.)

#447 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Explain the Testing instructions
To run this project you can run it using live server after opening the project in visual studio code.

**Test Configuration**:

- Operating system: Windows/Linux/macOs
- Version: any
- Text-editors used: Visual Studio Code

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
Screenshot 2021-04-11 105333
Screenshot 2021-04-11 105401
Screenshot 2021-04-11 105421
Screenshot 2021-04-11 105436
Screenshot 2021-04-11 105543
Screenshot 2021-04-11 105603

### CODE REVIEW

**Review by astha2412 — CHANGES_REQUESTED** [review]
Make it more responsive!
Screenshot (413)

**Comment by RajAdwaita** [comment]
Hi, I just made a fix , could you please take a look at it

**Comment by arpit456jain** [comment]
@RajAdwaita i think this PR contains some files of shoe related template made in react js

File: ShoeStore/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "pizza",
```
Comment (arpit456jain) [inline ShoeStore/package.json:2]: like this

Comment (RajAdwaita) [reply ShoeStore/package.json:2]: No bro this is pure HTML CSS js based work. I am sorry just noticed that I pushed a few changes that I made in the shoe store website also.

Comment (RajAdwaita) [reply ShoeStore/package.json]: I just made a push and removed the react based website. Could you please check it once noq

**Comment by RajAdwaita** [comment]
Could you please check it one now @arpit456jain

**Comment by arpit456jain** [comment]
there are lots of files needed to be removed
image

**Comment by RajAdwaita** [comment]
Okay I'll put all the svg files in a separate folder and make push again

**Comment by RajAdwaita** [comment]
@arpit456jain bro the screenshot that you just posted, those files are of another project which I am contributing to your repository itself.

**Review by arpit456jain — APPROVED** [review]

**Review by arpit456jain — APPROVED** [review]

**Comment by RajAdwaita** [comment]
Thank you and hoping to contribute more to this Fontend templates repository

**Pull request merged**